src/style.css

```diff
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: grey;
```

Comment:
```suggestion
  background-color: #ccc;
```
рекомендую через код

Reply:
Изменил, спасибо